tests/functional/lti_certification/v13/conftest.py

```diff
@@ -0,0 +1,173 @@
+import json
+from datetime import datetime
+
+import importlib_resources
+import jwt
+import pytest
+from httpretty import httpretty
+
+from tests import factories
+
+keys_path = importlib_resources.files("tests.functional.lti_certification.v13")
+JWT_KID = "JWT_KEY_KID"
+
+
+@pytest.fixture(scope="session")
+def jwt_private_key():
+    with open(keys_path / "jwt_private.key", encoding="utf8") as key:
+        return key.read()
+
+
+@pytest.fixture(scope="session")
+def jwt_public_key():
+    with open(keys_path / "jwt_public.key", encoding="utf8") as key:
+        return json.load(key)
+
+
+@pytest.fixture
+def jwt_headers():
+    return {"kid": JWT_KID}
+
+
+@pytest.fixture
+def make_jwt(jwt_headers, jwt_private_key):
```

Comment:
`make_jwt` uses our `jwt_private_key` to sign the JWTs

We have `jwk_endpoint_intercept` that would use the public key to decode them.